zombies.js

```diff
@@ -100,27 +94,29 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
- class Player{
-
-  constructor(name, health, strength, speed){
-    this.name = name;
+ function Player(name, health, strength, speed){
+    Item.call(this, name);
```

Comment:
Player should not be extending Item. I wouldn't think of a person as an item. Also, this would make Player a child of Item when they really are a separate family altogether.